src/components/ProtectedRoute.js

```diff
@@ -46,11 +48,51 @@ class ProtectedRoute extends Component {
             )
           )}
         />
+        <Divider section />
+        <div style={{ textAlign: 'center', margin: '10px 10% 10px 10%', paddingBottom: '20px' }}>
+          <p>
+            <FormattedMessage
+              id="app.protected_route.conduct_message"
+              description="Rovercode Code of Conduct"
```

Comment:
These descriptions exist to help the translator understand the meaning of the string he/she is translating. It's helpful to include more information here like `Description and link to inform the user of the location of the Rovercode Code of Conduct`